cslbot/helpers/textutils.py

```diff
@@ -369,11 +369,50 @@ def gen_intensify(msg):
     return "[%s INTENSIFIES]" % msg.upper()
 
 
+def gen_djones(msg: str) -> str:
```

Comment:
https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments

"On the other hand, never describe the code. Assume the person reading the code knows Python (though not what you're trying to do) better than you do."

You have way the hell too many comments here, and the only type annotations that actually add value are the ones in the method signature.